bitstore/controllers.py

```diff
@@ -51,6 +51,8 @@ def authorize(auth_token, req_payload):
             })
             try:
                 s3path = config['STORAGE_PATH_PATTERN'].format(**format_params)
+                # Avoid double slash if "dataset" is not in req_payload (as it is optional)
```

Comment:
Why would we get a double slash? I don't understand this special casing for dataset